src/PowerShellEditorServices/Hosting/HostStartupInfo.cs

```diff
@@ -0,0 +1,157 @@
+using System;
+using System.Collections.Generic;
+using System.Management.Automation.Host;
+
+namespace Microsoft.PowerShell.EditorServices.Hosting
+{
+    /// <summary>
+    /// Contains details about the host as well as any other information
+    /// needed by Editor Services at startup time.
+    /// </summary>
+    public class HostStartupInfo
+    {
+        #region Constants
+
+        /// <summary>
+        /// The default host name for PowerShell Editor Services.  Used
+        /// if no host name is specified by the host application.
+        /// </summary>
+        public const string DefaultHostName = "PowerShell Editor Services Host";
+
+        /// <summary>
+        /// The default host ID for PowerShell Editor Services.  Used
+        /// for the host-specific profile path if no host ID is specified.
+        /// </summary>
+        public const string DefaultHostProfileId = "Microsoft.PowerShellEditorServices";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/app/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4579423)